### PR TITLE
🎨 clean up footer

### PR DESCRIPTION
**Pull request by vikaspotluri123**
- move team to humans.txt, link in footer
- increase line-height for paragraphs
- move frame styles to css

I'm also thinking of creating a basic UI with the humans.txt content - thoughts?

### CODE REVIEW

**Comment by aslafy-z** [comment]
Doesn't look good on mobile (can't reproduce in the responsive view): 

img_20190126_142910

File: src/humans.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	Site: https://mihir.ch
+
+	Maintainer: Zadkiel
+	Site: https://github.com/aslafy-z
+
+	Maintainer: Vikas Potluri
+	Site: https://github.com/vikaspotluri123
+
+/* THANKS */
+
```
Comment (aslafy-z) [inline src/humans.txt]: Drop this NL or add one in the upper section

File: src/pug/includes/body.pug [hunk 2]
```diff
@@ -64,13 +64,15 @@ include ./fork.pug
 
 	footer.flex
 		.item.flex
-			iframe(src='https://upscri.be/67d2fa?as_embed', style='width:100%;max-width:800px;margin:0 auto;', height='400', frameborder='0', title='Subscribe for updates')
+			iframe(
+				src='https://upscri.be/67d2fa?as_embed',
+				class='subscribe-form'
+				height='400',
```
Comment (aslafy-z) [inline src/pug/includes/body.pug]: You can drop all the tailing commas

File: src/pug/includes/body.pug [hunk 3]
```diff
@@ -64,13 +64,15 @@ include ./fork.pug
...
+				frameborder='0',
+				title='Subscribe for updates'
+			)
 		.item.flex
-			p Made with ❤︎ by #[a(href="https://github.com/swapagarwal") Swapnil Agarwal]
-		.item.flex
-			p Designed by #[a(href="https://github.com/zac-garby") Zac Garby] and #[a(href="https://mihir.ch/") Mihir Chaturvedi]
-		.item.flex
-			p Maintained by #[a(href="https://github.com/aslafy-z") Zadkiel] and #[a(href="https://github.com/vikaspotluri123") Vikas Potluri]
+			p devSwag is made possible by #[a(href="/humans.txt") some wonderful humans]
```
Comment (aslafy-z) [inline src/pug/includes/body.pug]: I ❤︎ the `❤︎` emoji, maybe we can keep it?

**Comment by aslafy-z** [comment]
ping @vikaspotluri123

File: src/pug/includes/body.pug (hunk 3 above)
Comment (vikaspotluri123) [reply src/pug/includes/body.pug]: How would you suggest we do that (wording wise)? Maybe add it to the end of the sentence?

**Comment by ShivamGoyal1899** [comment]
The separate file seems like it's not required at first. The PR seems pretty dormant as well. Also, a design overhaul is already under progress #443

**Comment by vikaspotluri123** [comment]
Closing since #443 has been merged

**Pull request closed without merging**